src/Serialization/Rings.jl

```diff
@@ -495,15 +469,15 @@ end
 
 ################################################################################
 # Laurent Series
-@registerSerializationType(Generic.LaurentSeriesRing, true, "LaurentSeriesRing")
-@registerSerializationType(Generic.LaurentSeriesField, true, "LaurentSeriesField")
-@registerSerializationType(ZZLaurentSeriesRing)
+@register_serialization_type Generic.LaurentSeriesRing  "LaurentSeriesRing" uses_id
+@register_serialization_type Generic.LaurentSeriesField "LaurentSeriesField" uses_id
```

Comment:
Maybe we should add a comment which explains why the optional string argument is used here -- namely because the type otherwise would be serialized as `AbstractAlgebra.Generic.LaurentSeriesField` (I think)